homework_9/company.py

```diff
@@ -0,0 +1,84 @@
+# 1. Create a class describing any company. For example, Toshiba or Apple
+
+from random import choice
+
+
+class Company:
+    APPLE_REVENUE_2022 = 387.53
+    AVERAGE_STADIUM_CAPACITY = 100000
+
+    def __init__(self, name: str, revenue: float, key_people: list, total_employees: int, products: list = None,
+                 services: list = None,
+                 ):
+        self.__name = name
+        self.__revenue = revenue
+        self.__products = products
+        self.__services = services
+        self.__key_people = key_people
+        self.__total_employees = total_employees
+
+    @property
+    def name(self):
+        return self.__name
+
+    @property
+    def revenue(self):
+        return self.__revenue
+
+    @property
+    def key_people(self):
+        return self.__key_people
+
+    @property
+    def total_employees(self):
+        return self.__total_employees
+
+    @total_employees.setter
+    def total_employees(self, new_total: int):
+        self.__total_employees = new_total
```

Comment:
Looks like I can set a negative number for this property, or event smth that is not integer